docs/source/algorithms/SymmetriseMDHisto-v1.rst

```diff
@@ -0,0 +1,45 @@
+.. algorithm::
+
+.. summary::
+
+.. relatedalgorithms::
+
+.. properties::
+
+Description
+-----------
+
+Function to symmeterise an :ref:`MDHistoWorkspace <MDHistoWorkspace>` with 3 dimensions (HKL) using the
+symmetry operations of the point group of the Laue class (inferred from the ``PointGroup`` or ``Spacegroup`` provided).
+
+Two types of average can be performed:
+
+    1. ``WeightedAverage=False`` (simple mean of the signal values and errors are added in quadrature then divided by the number of contributing bins)
+    2. ``WeightedAverage=True`` (weighted average performed using inverse square of errors as weights).
+
+If (2) ``WeightedAverage=True`` then the signal is given by :math:`s = \Sigma_i w_i s_i` where
+:math:`s_i` is the signal in symmetrically equivalent bin :math:`i` and the weights are the inverse of the
+error squared in each bin, :math:`w_i = 1/e_i^2`. The final error is given by :math:`1/e^2 = \Sigma_i w_i`.
+
+
+Usage
+-----
+
+**Example:**
+
+.. code-block:: python
+
+    from mantid.simpleapi import *
+
+    y = np.ones(8)
+    nbins = int(len(y) ** (1 / 3))
+    ws = CreateMDHistoWorkspace(SignalInput=y, ErrorInput=y,
+                                     Dimensionality=3, Extents='-1,1,-1,1,-1,1',
+                                     NumberOfBins=3 * [nbins], Names='H,K,L',
+                                     Units='rlu,rlu,rlu')
+
+    ws_sym = SymmetriseMDHisto(ws, PointGroup="-1", WeightedAverage=False)
```

Comment:
If the algorithm doesn't take too long to run (and if it does, there should be some progress reporting added to it), it's best if we can use the `.. testcode:: ...` directive here to make sure that this code gets run by the doctests and is flagged if it ever stops working.

Reply:
I think I should add progress reporting actually. 
I can make it a doc test, is that necessary given I think this case is already covered in a unit test?

Reply:
It's still valuable in case the algorithm's API changes. It's not necessarily a test of the algorithm itself, just confirms that these instructions in our documentation still work (such as after a python or API upgrade, for example).